packages/@okta/vuepress-theme-prose/components/Dialog.vue

```diff
@@ -0,0 +1,131 @@
+            Workforce Identity Cloud
+          </p>
+          <h4 class="c-dialog__block-title dont-break-out">
+            My employees, <br> contractors, and partners
+          </h4>
+          <p class="c-dialog__block-text c-dialog__paragraph dont-break-out">
+            Workforce Identity Cloud powers identity for all employees, business partners, and contractors and ensures they have access to what they need, when they need it with the appropriate levels of privilege.
+          </p>
+          <button
+            class="c-dialog__link c-dialog__link_type_primary dont-break-out"
+            type="button"
+          >
+            Go to Workforce Identity Cloud docs
+          </button>
+        </article>
+        <article class="c-dialog__block c-dialog__block_type_secondary">
+          <div class="c-dialog__block-wrapper">
+            <p class="c-dialog__block-paragraph dont-break-out">
+              Customer Identity Cloud
+            </p>
+            <p class="c-dialog__block-wrapper-text dont-break-out">
+              Powered by
+            </p>
+            <img
+              class="c-dialog__block-image"
+              src="/img/dialog/auth.svg"
+              width="70"
+              height="29"
+              alt="Auth0"
+            >
+          </div>
+          <h4 class="c-dialog__block-title dont-break-out">
+            My customers <br> or SaaS applications
+          </h4>
+          <p class="c-dialog__block-text c-dialog__paragraph dont-break-out">
+            Customer Identity Cloud enables app builders, digital marketers, and security teams to give end-users access to everything they need online, in a convenient, secure way.
+          </p>
+          <a
+            class="c-dialog__link dont-break-out"
+            href="https://developer.auth0.com"
+          >
+            Go to Customer Identity Cloud docs <span aria-hidden="true">↗</span>
+          </a>
+        </article>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+  export default {
+    name: "HomeDialog",
+    mounted() {
+      const dialog = document.querySelector(".c-dialog");
+      const dialogClose = dialog.querySelector(".c-dialog__close");
+      const dialogOverlay = dialog.querySelector(".c-dialog__overlay");
+      const dialogButton = dialog.querySelector(".c-dialog__link_type_primary");
+
+      dialogClose.addEventListener("click", this.hideDialog);
+      dialogOverlay.addEventListener("click", this.hideDialog);
+      dialogButton.addEventListener("click", this.hideDialog);
+
+      window.addEventListener("keydown", (event) => {
+        if (event.keyCode === 27) {
+          if (!dialog.classList.contains("c-dialog_state_hidden")) {
+            this.hideDialog();
+          }
+        }
+      })
+    },
+    methods: {
+      hideDialog() {
+        const dialog = document.querySelector(".c-dialog");
+
+        dialog.classList.add("c-dialog_state_hidden");
```

Comment:
Instead of using `document.querySelector` we can use a state variable like `dialogHidden` that is set in this method to show/hide the element.